src/contrib/mrtgindex.cgi/mrtgindx.cgi

```diff
@@ -71,11 +71,11 @@
 
 #-------------------------------------------------------
 # Modify this statement to match your configuration
-@config_files = ('mrtg.cfg'); # Single config file
+@config_files = ('/etc/mrtg.cfg'); # Single config file
```

Comment:
same problem as before ... it should also e possible to run mrtg as a normal user ...